app/user/user.sidebar.js

```diff
@@ -1,28 +1,62 @@
 (function () {
   'use strict';
 
+  var MY_ACCOUNT = {
+    text: "My Account",
+    sref: "app.account.settings",
+  };
+
+  var SUB_CLIENTS = {
+    text: "Sub Clients",
+    sref: "app.user.client.sub.list",
+  };
+
+  var SUPER_CLIENTS = {
+    text: "Super Clients",
+    sref: "app.user.client.super.list",
+  };
+
   angular
     .module('app.user')
-    .config(NavConfig)
+    .run(UserSidebarShowWithPermissions)
     ;
 
   /**
    * @ngInject
    */
-  function NavConfig(NavProvider) {
-    NavProvider.group('user', {
-      translate: "nav.USERS",
-      sref: "app.account.settings",
-      icon: "fa fa-user",
-    }).item({
-      text: "My Account",
-      sref: "app.account.settings",
-    }).item({
-      text: "Sub Clients",
-      sref: "app.user.client.sub.list",
-    }).item({
-      text: "Super Clients",
-      sref: "app.user.client.super.list"
-    });
+  function UserSidebarShowWithPermissions(Auth, Select /*, Permission*/) {
```

Comment:
permission can't be imported because this is the client side. Clients don't have permissions - only Admins & Integrations do. We need a way of getting this information off of the client account, e.g. `GET /client/{authed ID}`.